src/dodal/devices/attenuator/filter_selections.py

```diff
@@ -0,0 +1,72 @@
+from ophyd_async.core import SubsetEnum
+
+
+class p99StageSelections(SubsetEnum):
```

Comment:
```suggestion
class P99FilterSelections(SubsetEnum):
```

Reply:
I think if we can make these enums consistently named then I think it's ready to merge